tools/schema_check.sh

```diff
@@ -6,10 +6,13 @@
 
 set -e
 
+BUILD_TOOL_SCHEMAS_VERSION=latest
```

Comment:
We should be careful to not break the spec workflows if new images are published that break existing behavior. I'd prefer pinning the version.

Reply:
There is currently no tagged version of the tool docker image. I don't know how to make a release of https://github.com/open-telemetry/build-tools so that we have a tagged version. Do you know?

Reply:
Good point, this is something that should be documented there. One just has to create Github release and add a tag in the same format as the others and it will push all containers with the version specified in the tag. Ideally we'd merge https://github.com/open-telemetry/build-tools/pull/85 and then release both at once as `v0.11.0`.

Reply:
How about we merge this PR now and then we can pin to 0.11.0 when it is released? Alternatively we can release 0.11.0 now without waiting for https://github.com/open-telemetry/build-tools/pull/85

Reply:
```suggestion
BUILD_TOOL_SCHEMAS_VERSION=0.11.0
```

There you go: https://hub.docker.com/r/otel/build-tool-schemas/tags
https://github.com/open-telemetry/build-tools/releases/tag/v0.11.0

Reply:
Thanks. Pinned to 0.11.0.